cmd/interlock/run.go

```diff
@@ -35,7 +35,7 @@ var cmdRun = cli.Command{
 		cli.StringFlag{
 			Name:  "config, c",
 			Usage: "path to config file",
-			Value: "",
+			Value: "/etc/interlock/config.toml",
```

Comment:
As mentioned in another line, this will break the env var parsing.  Since we don't include a default I think we should remove this an error if no config is found.